policies/templates/gcp_gke_restrict_client_auth_methods_v1.yaml

```diff
@@ -0,0 +1,95 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Restrict client auth methods on GKE. 
+# https://cloud.google.com/kubernetes-engine/docs/how-to/hardening-your-cluster#restrict_authn_methods
+
+apiVersion: templates.gatekeeper.sh/v1alpha1
+kind: ConstraintTemplate
+metadata:
+  name: gcp-gke-restrict-client-auth-methods
+spec:
+  crd:
+    spec:
+      names:
+        kind: GCPGKERestrictClientAuthenticationMethodsConstraintV1
+        plural: GCPGKERestrictClientAuthenticationMethodsConstraintsV1
```

Comment:
let's update the plural name to be all lowercase

Reply:
updated